app/services/team_importer.rb

```diff
@@ -202,7 +190,7 @@ def update_smart_annotations(team)
   def update_annotation(text)
     return false if text.nil?
     updated = false
-    %w(prj exp tsk sam).each do |name|
+    %w(prj exp tsk).each do |name|
```

Comment:
You also need to include `rep_item` here; then the import should finally work as intended!